### ISSUE TITLE
Feature Request: Replace "Kids" Display in "People" tab with top tags

### ISSUE THREAD

**Issue by xasos** [monicahq/monica#451]
The "0 kids" here is strange, and feels a bit misplaced. Instead, it should have the top 3 (or so) tags associated with the contact.

![Contact List](http://i.imgur.com/Q4clgwf.jpg)

**Comment by Lovatt** [comment]
I agree the kids counter here feels like a stat for stats sake, there is no filtering or sorting by kids on this page, so why is it any more relevant than debts or gifts.

Tags are already on the right hand side of this page though, so my vote is just remove the kids counter 👍
